docs/_pages/installation/roborock.md

```diff
@@ -80,8 +80,19 @@ Example: https://dustbuilder.xvm.mit.edu/pkg/s5/v11_002008.fullos.fd043420-6ddb-
 ### Image Building
 It is recommended to use [https://github.com/zvldz/vacuum](https://github.com/zvldz/vacuum) to build the image.
 
-`--valetudo-path` expects a path to a folder containing a binary named `valetudo`.
-Refer to [https://github.com/Hypfer/Valetudo/releases](https://github.com/Hypfer/Valetudo/releases) to fetch the latest valetudo binary.
+`--valetudo-path` expects a path to a folder containing two things:
+
+ * The source code of [Valetudo](https://github.com/Hypfer/Valetudo)
+ * And a binary named `valetudo`.  Refer to [https://github.com/Hypfer/Valetudo/releases](https://github.com/Hypfer/Valetudo/releases) to fetch the latest valetudo binary.
+
+You can create a folder will all the needed things with the commands like:
+(but make sure to use the latest version from the [releases](https://github.com/Hypfer/Valetudo/releases) page)
+
+```
+git clone https://github.com/Hypfer/Valetudo.git /tmp/Valetudo
+cd /tmp/Valetudo
+wget https://github.com/Hypfer/Valetudo/releases/download/0.5.1/valetudo
```

Comment:
You can use `https://github.com/Hypfer/Valetudo/releases/latest/download/valetudo` and can then drop the note about fetching the latest version.

Reply:
Also drop the parts about /tmp. Users should be free to choose where they want to download. Same for the change below.

Reply:
Thank you for the `lastest` link, I didn't know about that link. I have used that link and dropped the note.

Reply:
Ok. I have removed the `/tmp`